docs/metadata.md

```diff
@@ -1,10 +1,11 @@
 # Metadata Description File
 
-A device's `metadata.json` file provides the source template for what the
-device is _supposed_ to be. It is used to provide the basic information for
-registering devices in a IoT Core registry, generation of other template
-files (e.g. _config_ block) associated with the device, and as a means for
-runtime validation of what the device is actually sending.
+A device's `metadata.json` is a description about the device - a specification
```

Comment:
pedantic grammar nit: Should be "about the device: a specification"  (colon rather than dash)